### ISSUE TITLE
Fillable PDFs generated from HTML form seem to stop working correctly after 0.12.3

### ISSUE THREAD

**Issue by aldefouw** [zakird/wkhtmltopdf_binary_gem#70]
**Issue description**
When generating fillable PDF from an HTML form, the fields are not editable via Adobe Acrobat in versions >0.12.3.

**Expected or desired behavior**
That fields are fillable in the PDF document.

**System specifications**
wkhtmltopdf 0.12.3 (with patched qt)

**platform/distribution and version :**
MacOSX 10.15.x

My current solution is to pin the version to 0.12.3.

**Comment by aldefouw** [comment]
When I ran a diff tool on the headers of the files that are output from the version that has the expected behavior vs. a newer version that fails to output editable fields.  , 

Here's the main difference I found:

**0.12.3.x**
```
/Parrent 8 0 R\
```

**0.12.4.0**
```
/Parent 8 0 R\
```


Appears that "Parrent" was renamed to "Parent" for newer versions.  

I can't find "Parrent" in the PDF spec, though ... so was the old version working only because of a mistake?  Hmm.
